I have no idea why INSTALL_PAGES dependent-set didn't work

--- CMakeLists.txt.orig	2020-09-19 06:48:47 UTC
+++ CMakeLists.txt
@@ -74,7 +74,7 @@ if (NOT ENABLE_CPU_CLIP)
 	set (CPU_CLIPS_NEGATIVE FALSE)
 endif ()
 cmake_dependent_option (ENABLE_COMPATIBLE_LIBSNDFILE_NAME "Set DLL name to libsndfile-1.dll (canonical name), sndfile.dll otherwise" OFF "WIN32;BUILD_SHARED_LIBS" OFF)
-cmake_dependent_option (INSTALL_MANPAGES "Install man pages for programs" ON "BUILD_PROGRAMS AND (UNIX OR MINGW OR CYGWIN)" OFF)
+option (INSTALL_MANPAGES "Install man pages for programs" ON)
 
 set (HAVE_EXTERNAL_XIPH_LIBS ${ENABLE_EXTERNAL_LIBS})
 set (HAVE_SQLITE3 ${BUILD_REGTEST})
